examples/getting_started/pendulum.py

```diff
@@ -19,7 +21,61 @@
     InitialGuess,
     ObjectiveFcn,
     Objective,
+    PlotType,
+    ObjectiveList,
 )
+def plot_objectives(ocp):
+
+    # TODO : Add a label tag to CustomPlot for the legend on the graph
+
+    number_of_plots = 0
+    objective_names = []
+    same_objectives = [[], []]
+    number_of_same_objectives = 0
+    for nlp in ocp.nlp:
+        for j in nlp.J:
+            if j.name in objective_names:
+                same_objectives[0].append(objective_names.index(j.name))
+                same_objectives[1].append(number_of_plots)
+                number_of_same_objectives += 1
+            else:
+                objective_names.append(j.name)
+            number_of_plots += 1
```

Comment:
Please make a subfunction of this section (just add `def ...` before and indent) and call the function. Like this, you have a neat way to comment the code without actually adding a comment (the function name will be used for that). Also this prevent from propagating temporary variable names that won't be useful later